content/data-sharing/mutable-and-sendable.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Mutable and Sendable"
+section: "Data Sharing"
+menu:
+  toc:
+    parent: "data-sharing"
+    weight: 40
+---
+
+## Problem
+
+Suppose you have a central Supervisor actor that manages a very large data structure that contains the work produced by a collection of Worker actors.  However, the Superviser must also respond to data requests (perhaps from the Workers themselves, or in this case, from another set of Requester actors).  As a result, the Superviser's data structure must be both mutable (to update it in response to feedback from Workers) and sendable (to respond to requests from Requesters).  `iso` is the only refcap that accommodates that, but our Supervisor can't use `iso` here because it must keep a reference to the data structure to continue to respond to incoming updates and requests.  How to proceed?
+
+## Solution
+
+```pony
+use mut = "collections"
+use "persistent/collections"
+
+type Datum is List[I32]
+
+
+actor Worker
+  let _supervisor: Supervisor
+
+  new create(supervisor: Supervisor) =>
+  _supervisor = supervisor
+
+  be do_work(id: USize) =>
+    let result = [as I32: 1; 2; 3; 4] // initial state array
```

Comment:
on small screens, this side comment is hard to deal with, can you move it above the `let result`. The same comment would apply elsewhere.